tests/conftest.py

```diff
@@ -22,6 +22,13 @@
     uvloop = None
 
 
+@pytest.fixture
+def disable_gc():
+    gc.disable()
+    yield
+    gc.enable()
```

Comment:
Let's not touch gc if it's already disabled globally:
```suggestion
    gc_is_on = gc.isenabled()

    if gc_is_on:
        gc.disable()
        gc.collect()

    yield

    if gc_is_on:
        gc.collect()
        gc.enable()
```

Reply:
👍 fixed